package.json

```diff
@@ -633,46 +627,23 @@
             "description": "Editor server default port",
             "default": 9000
           },
-          "dataEditorLogFile": {
-            "type": "string",
-            "description": "Path to log file for data editor",
-            "default": "${workspaceFolder}/dataEditor-${omegaEditPort}.log"
-          },
-          "dataEditorLogLevel": {
-            "type": "string",
-            "description": "Log level for data editor",
-            "enum": [
-              "error",
-              "warn",
-              "info",
-              "debug",
-              "trace"
-            ],
-            "default": "info"
-          },
-          "dfdlDebuggerLogLevel": {
-            "type": "string",
-            "description": "Log level for debugger",
-            "enum": [
-              "ALL",
-              "DEBUG",
-              "ERROR",
-              "INFO",
-              "OFF",
-              "TRACE",
-              "WARN"
-            ],
-            "default": "INFO"
-          },
-          "dfdlDebuggerLogFile": {
-            "type": "string",
-            "description": "Path to log file for debugger",
-            "default": "/tmp/daffodil-debugger.log"
+          "dataEditor": {
+            "type": "object",
+            "description": "Configuration for Data Editor. Settings are port, logFile and logLevel",
+            "default": {
+              "port": 9000,
+              "logFile": "${workspaceFolder}/dataEditor-${omegaEditPort}.log",
+              "logLevel": "info"
+            }
```

Comment:
I prefer the way this is done in the `dfdlDebugger` section, so make this:

```
           "default": {
              "port": 9000,
              "logging": {
                "file": "${workspaceFolder}/dataEditor-${omegaEditPort}.log",
                "level": "INFO"
              }
            }
```

So we don't have 2 different ways to do the same thing.